Source/JavaScriptCore/runtime/JSGlobalObject.h

```diff
@@ -539,6 +539,8 @@ class JSGlobalObject : public JSSegmentedVariableObject {
     bool m_webAssemblyEnabled { true };
     bool m_needsSiteSpecificQuirks { false };
     unsigned m_globalLexicalBindingEpoch { 1 };
+    ScopeOffset m_lastStaticGlobalOffset;
+    HashSet<UniquedStringImpl*> m_varNamesDeclaredViaEval;
```

Comment:
Should we use IdentifierSet?

Reply:
Thanks Yusuke, switched to `IdentifierSet`.